unit-tests/sw-dev/test-units.py

```diff
@@ -0,0 +1,55 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+import pyrealsense2 as rs
+from rspy import log, test
+import sw
+
+    
+with sw.sensor( "Stereo Module" ) as sensor:
+    depth = sensor.video_stream( "Depth", rs.stream.depth, rs.format.z16 )
+    sensor.start( depth )
+
+    with test.closure( "By default, frames do not have units" ):
+        f = depth.frame()
+        test.check_equal( f.depth_units, 0. )
```

Comment:
When we ask for the frame itself, does the SW sensor multiple each pixel with the depth unit?

Reply:
This line is the depth-units in the SW frame **input**. Before my fix, this was some uninitialized value, so random. Now it's 0.